api/api/src/main/resources/application.conf

```diff
@@ -0,0 +1,7 @@
+es.host=${?es_host}
```

Comment:
We setting these properly in terraform?

Reply:
We appear to be setting these in the terraform here:

https://github.com/wellcometrust/catalogue/blob/ea151ecb666d2fc644bf8fdf72a01ac4ed8c2e12/api/terraform/catalogue_api/services/service/main.tf#L60-L66

I don't really understand how these are actually resolved into the correct envvars  though.

Reply:
They are sent to ECS as references, which are then looked up from secrets store, and pushed in as ENV vars on container startup.

@kenoir as far as I know this is the correct way.